lint/rule_panel_title_description_test.go

```diff
@@ -0,0 +1,43 @@
+package lint
+
+import (
+	"testing"
+)
+
+func TestPanelTitleDescription(t *testing.T) {
+	linter := NewPanelTitleDescriptionRule()
+
+	for _, tc := range []struct {
+		result Result
+		panel  Panel
+	}{
+		{
+			result: Result{
+				Severity: Error,
+				Message:  "Dashboard 'test', panel with id '1' has missing title or description, currently has title '' and description: ''",
```

Comment:
Can you add tests for both failure cases as well? So, a test with a title, but no description, and a test with the description, but no title.